cmd/manager/main.go

```diff
@@ -39,7 +40,16 @@ func main() {
 	flag.BoolVar(&printVersion, "version", false, "print version and exit")
 
 	klog.InitFlags(nil)
-	watchNamespace := flag.String("namespace", "", "Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.")
+	watchNamespace := flag.String(
+		"namespace",
+		"",
+		"Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.",
+	)
+	healthAddr := flag.String(
+		"health-addr",
+		":9440",
```

Comment:
why did we choose this default?

Reply:
Just a number similar to upstream implementation. There is no standard: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1487/files#r367928052